ui/src/components/tree/QTree.js

```diff
@@ -44,6 +44,7 @@ export default createComponent({
 
     color: String,
     controlColor: String,
+    keepColor: { type: Boolean, default: true },
```

Comment:
Not on the same line.

```suggestion
    keepColor: {
      type: Boolean,
      default: true
    },
```